test/box-luatest/gh_6539_log_user_space_empty_or_nil_select_test.lua

```diff
@@ -16,9 +16,22 @@ end)
 local expected_log_entry = 'C> empty or nil `select` call on user space ' ..
                            'with id=%d+'
 local grep_log_bytes = 256
-local dangerous_call_fmts = {'box.space.%s:select()', 'box.space.%s:select{}',
-                             'box.space.%s:select(nil, {})',
-                             'box.space.%s:select(nil, {fullscan = false})'}
+local dangerous_call_fmts = {
+    'box.space.%s:select(nil)',
+    'box.space.%s:select({})',
+    'box.space.%s:select(nil, {})',
+    'box.space.%s:select(nil, {iterator = "EQ"})',
+    'box.space.%s:select(nil, {iterator = "REQ"})',
+    'box.space.%s:select(nil, {fullscan = false})',
+    'box.space.%s:select(nil, {limit = 1001})',
+    'box.space.%s:select(nil, {offset = 1})',
+    'box.space.%s:select(nil, {limit = 500, offset = 501})',
+    'box.space.%s:select({0}, {limit = 1001, iterator = "ALL"})',
+    'box.space.%s:select({0}, {limit = 1001, iterator = "GE"})',
+    'box.space.%s:select({0}, {limit = 1001, iterator = "GT"})',
+    'box.space.%s:select({0}, {limit = 1001, iterator = "LE"})',
+    'box.space.%s:select({0}, {limit = 1001, iterator = "LT"})',
+}
```

Comment:
AFAICS this test doesn't check the `select_luac` method, only `select_ffi`. You can either call `select_luac` directly or make this test multi-engine (`*_luac` methods are used in Vinyl). In the latter case, the test should probably be moved to `engine-luatest`.

Reply:
It does:
https://github.com/CuriousGeorgiy/tarantool/blob/fc410b3c49de05c0dc3acf128f648a38a1660d35/test/box-luatest/gh_6539_log_user_space_empty_or_nil_select_test.lua#L49-50

At the time I was writing the test I didn't know about the _test group parametrization_ feature in **luatest**.